bench/benchmark.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var benchmark = require('benchmark');
+var Table = require('cli-table');
+var colors = require('colors/safe');
+
+var oldZ = require('sanctuary-type-classes');
+var newZ = require('..');
+
+
+function identity(x) {
+  return x;
+}
+
+function formatPct(pct) {
+  return ('+' + pct.toLocaleString('en', {
+    minimumIntegerDigits: 3,
+    minimumFractionDigits: 1,
+    maximumFractionDigits: 1,
+    useGrouping: false
+  })).slice(-6) + '%';
+}
+
+function format(res) {
+  return res.hz.toLocaleString('en', {maximumFractionDigits: 0}) + ' Hz '
+       + '±' + res.stats.rme.toFixed(2) + '%'
+       + '(n ' + String(res.stats.sample.length) + ')';
+}
+
+module.exports = function runBenchmarks(specs) {
+  var table = new Table({head: ['name', 'Old', 'New', 'change']});
```

Comment:
`Old` and `New` are not particularly descriptive. Would it be possible to show something like `v6.0.0` and `avaq/untangle@abc1234` instead? This way the table would not require context in order to be meaningful, so it could be shared more easily (on Gitter, for example).